### PR TITLE
Chat Visual fixes 1

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] pointer on image attachments. use method arrows and not inline arrows
- [x] render semi-markdown in the inbox
- [x] usernames in chat header send you to the profile screen

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers first round of chat fixes suggested by @cecileboucheron

File: shared/chat/conversation/messages/attachment.desktop.js [hunk 1]
```diff
@@ -164,19 +164,19 @@ function AttachmentMessageGeneric ({message, onOpenInFileUI}: {message: Constant
...
 }
 
 export default class AttachmentMessage extends PureComponent<void, Props, void> {
-  _onOpenInPopup () {
+  _onOpenInPopup = () => {
     this.props.onOpenInPopup(this.props.message)
   }
 
-  _onOpenInFileUI () {
+  _onOpenInFileUI = () => {
```
Comment (MarcoPolo) [inline shared/chat/conversation/messages/attachment.desktop.js]: thanks, I was meaning to come back to this

Comment (MarcoPolo) [inline shared/chat/conversation/messages/attachment.desktop.js]: @chromakode this is the pattern I was talking about before

File: shared/chat/conversation/messages/attachment.desktop.js [hunk 2]
```diff
@@ -187,10 +187,10 @@ export default class AttachmentMessage extends PureComponent<void, Props, void>
     let attachment
     switch (message.previewType) {
       case 'Image':
-        attachment = <AttachmentMessagePreviewImage message={message} onOpenInPopup={() => this._onOpenInPopup()} onOpenInFileUI={() => this._onOpenInFileUI()} />
+        attachment = <AttachmentMessagePreviewImage message={message} onOpenInPopup={this._onOpenInPopup} onOpenInFileUI={() => this._onOpenInFileUI()} />
```
Comment (MarcoPolo) [inline shared/chat/conversation/messages/attachment.desktop.js]: this should be `onOpenInFileUI={this._onOpenInFileUI}`

**Pull request merged**